rust/tw_proto/build.rs

```diff
@@ -0,0 +1,47 @@
+use pb_rs::types::FileDescriptor;
```

Comment:
I wonder if we can create file template in Clion to insert license headers

Reply:
Right, always forget about the headers...

Reply:
Fixed